support/proxy/vwf.example.com/kinetic/drawing.js

```diff
@@ -52,9 +52,6 @@ this.clientLeave = function( moniker ) {
         delete this.drawing_clients[ moniker ]; 
         this.drawing_clients = this.drawing_clients;
     }
-    if ( this.drawing_private[ moniker ] !== undefined ) {
-        delete this.drawing_private[ moniker ]; 
-    }
 };
 
-
+//# sourceURL=drawing.js
```

Comment:
One annoying detail of the model-side functions is that each one of them actually needs the `sourceURL` inside them.  Having the main one for the file will allow the file to show up in the debugger, but it's not actually that "version" of the code that gets executed so breakpoints won't get hit inside it.  I can explain more in person, if you're interested.